packages/server-wallet/src/db/migrations/20201211180815_challenge_status.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
+
+const tableName = 'challenge_status';
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable(tableName, function(table) {
+    table.string('channel_id').notNullable();
+    table.integer('finalizes_at').notNullable();
+    table.integer('block_number').notNullable();
+
+    table.primary(['channel_id']);
+    table.foreign('channel_id').references('channels.channel_id');
```

Comment:
Did you consider the choice of new columns on the existing table versus an entirely new table? Any thoughts on the tradeoffs there?

Reply:
(Just spotted existing discussion under #3070)